UtopiaRDS/utopiaAirlineSchema.sql

```diff
@@ -0,0 +1,141 @@
+-- MySQL dump 10.13  Distrib 8.0.16, for Win64 (x86_64)

```

Comment:
I may address this in a future PR (and so will still 👍 this PR), but it would be better to strip out all the MySQL configuration metadata comments ...

Reply:
Yes, I agree!